### PR TITLE
docs: backport guide improvements

### PR DESCRIPTION
**Pull request by jodh-intel**
Add more detail to the backport guide including some ASCII graphics to demonstrate backporting and formatting improvements.

Fixes: #110.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

File: Backport-Guide.md [hunk 1]
```diff
@@ -1,23 +1,96 @@
...
+    * [Raise a PR](#raise-a-pr)
 
-This document provides a short guide on how to perform a stable backport of a
-PR or patchset in the Kata Containers repositories.
+## Introduction
 
-It does not cover all eventualities that might be encountered while performing
-a backport, and only documents one potential Git based workflow. It does
-serve as a short introduction and reminder of the basic stable backport process.
+This document provides a short guide which explains how to perform a backport.
```
Comment (grahamwhaley) [inline Backport-Guide.md]: s/which explains/explaining/ ?

File: Backport-Guide.md [hunk 2]
```diff
@@ -26,77 +99,86 @@ into the `stable-1.2` branch of the `runtime` repo.
     $ git fetch origin
     ```
 
-    And check the list of current stable branches:
+    Check the list of stable branches:
 
     ```bash
     $ cd ${GOPATH}/src/github.com/kata-containers/runtime
```
Comment (grahamwhaley) [inline Backport-Guide.md]: I know you didn't add it, but this second `cd` is superfluous I think.

File: Backport-Guide.md [hunk 3]
```diff
@@ -26,77 +99,86 @@ into the `stable-1.2` branch of the `runtime` repo.
...
+  ```bash
+  $ git cherry-pick -x <commit>...
+  ```
+
+  You can cherry pick ranges of commits. Please see the `git-cherry-pick(1)`
+  man page for more information.
 
-    You can cherry pick ranges of commits, etc. Please see the `git-cherry-pick(1)`
-    man page for more information.
+  > *Note:* You do not need to open a new `Issue` for or add an extra `Fixes: nnn` item
```
Comment (grahamwhaley) [inline Backport-Guide.md]: s/for or add/or add/ - maybe - the `for` makes the sentence read funny to me.

File: Backport-Guide.md [hunk 4]
```diff
@@ -26,77 +99,86 @@ into the `stable-1.2` branch of the `runtime` repo.
...
-    of that branch.
+To list all commits in the `master` branch which are not in the `stable-1.2`
+branch:
 
-    ```bash
-    $ git log --oneline master
-    # And search for the SHA of the commits you wish to backport.
-    ```
+```bash
+$ git log --oneline --no-merges ..master
```
Comment (grahamwhaley) [inline Backport-Guide.md]: thanks for adding the `--no-merges`. Worth making a formal note in the document body that we do *not* want merge commits cherry picked over?

File: Backport-Guide.md (hunk 1 above)
Comment (jodh-intel) [reply Backport-Guide.md]: Fixed.

File: Backport-Guide.md (hunk 2 above)
Comment (jodh-intel) [reply Backport-Guide.md]: Agreed - fixed.

File: Backport-Guide.md (hunk 3 above)
Comment (jodh-intel) [reply Backport-Guide.md]: I think that's the original wording, but agreed - fixed ;)

File: Backport-Guide.md (hunk 4 above)
Comment (jodh-intel) [reply Backport-Guide.md]: Agreed - updated.

**Comment by jodh-intel** [comment]
Hi @klynnrif - ptal.

**Comment by jodh-intel** [comment]
Ping y'all! I'd really like to get this landed... ;)

**Pull request merged**